Client/Frontend/Components/FakespotErrorCardView.swift

```diff
@@ -0,0 +1,143 @@
+        label.textColor = .white
+        label.font = DefaultDynamicFontHelper.preferredFont(
+            withTextStyle: .footnote,
+            size: UX.buttonFontSize)
+        label.numberOfLines = 0
+        label.setContentCompressionResistancePriority(.required, for: .vertical)
+        label.accessibilityIdentifier = AccessibilityIdentifiers.Components.ErrorCard.description
+    }
+
+    private lazy var primaryButton: ResizableButton = .build { button in
+        button.titleLabel?.font = DefaultDynamicFontHelper.preferredBoldFont(
+            withTextStyle: .callout,
+            size: UX.buttonFontSize)
+        button.layer.cornerRadius = UX.buttonCornerRadius
+        button.titleLabel?.textAlignment = .center
+        button.addTarget(self, action: #selector(self.primaryAction), for: .touchUpInside)
+        button.titleLabel?.adjustsFontForContentSizeCategory = true
+        button.contentEdgeInsets = UIEdgeInsets(top: UX.buttonVerticalInset,
+                                                left: UX.buttonHorizontalInset,
+                                                bottom: UX.buttonVerticalInset,
+                                                right: UX.buttonHorizontalInset)
+        button.accessibilityIdentifier = AccessibilityIdentifiers.Components.ErrorCard.primaryAction
+    }
+
+    /// Custom init method to pass title, description text, action text
+    init(title: String, description: String, actionTitle: String) {
+        super.init(frame: .zero)
+        setupLayout()
+        titleLabel.text = title
+        descriptionLabel.text = description
+        primaryButton.setTitle(actionTitle, for: .normal)
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+        setupLayout()
+    }
+
+    private func setupLayout() {
+        layer.cornerRadius = 4
+        addSubview(containerStackView)
+        NSLayoutConstraint.activate([
+            iconImageView.heightAnchor.constraint(equalToConstant: 24),
+            iconImageView.widthAnchor.constraint(equalToConstant: 24),
+
+            containerStackView.leadingAnchor.constraint(equalTo: leadingAnchor),
+            containerStackView.trailingAnchor.constraint(equalTo: trailingAnchor),
+            containerStackView.topAnchor.constraint(equalTo: topAnchor),
+            containerStackView.bottomAnchor.constraint(equalTo: bottomAnchor)
+            ])
+    }
+
+    @objc
+    private func primaryAction() {
+        // Add your button action here
+    }
+
+    // MARK: - ThemeApplicable
+    func applyTheme(theme: Theme) {
+        titleLabel.textColor = theme.colors.textInverted
+        descriptionLabel.textColor  = theme.colors.textInverted
+        iconImageView.tintColor = theme.colors.textInverted
+
+        primaryButton.setTitleColor(theme.colors.textInverted, for: .normal)
+        primaryButton.backgroundColor = UIColor(rgb: 0x961D1B)
+
+        layer.backgroundColor = UIColor(rgb: 0xC5292D).cgColor
+    }
```

Comment:
Posted a question on the design doc. I can't seem to find the hex in the Colors doc. Some help would be much appreciated.

Reply:
We are not using hex colors. All colors should be colors from our theme.